apollo-server/v3/integration-testing/src/main.ts

```diff
@@ -0,0 +1,7 @@
+import { createApolloServer } from './server';
+
+const createServer = async () => {
```

Comment:
I don't think this adds much over just calling `createApolloServer` directly?

Reply:
I suppose not, good point! Removing that extra function,